### PR TITLE
sql: update CREATE USER password description

### PR DESCRIPTION
**Pull request by lilin90**
@tiancaiamao @CaitinChen PTAL
Via: https://github.com/pingcap/docs-cn/pull/969

### CODE REVIEW

File: sql/privilege.md [hunk 1]
```diff
@@ -324,10 +324,10 @@ auth_spec: {
...
+- `IDENTIFIED BY 'auth_string'`
 
-  When you set the login password, `auth_string` is encrypted by TiDB and stored in the `mysql.user` table.
+    It is used to set the login password. `auth_string` is encrypted by TiDB and stored in the `mysql.user` table.
 
-- IDENTIFIED BY PASSWORD `hash_string`
+- `IDENTIFIED BY PASSWORD 'hash_string'`
 
-  When you set the login password, `hash_string` is encrypted by TiDB and stored in the `mysql.user` table. Currently, this is not the same as MySQL.
+    It is also used to set the login password. `hash_string` is a 41-bit string similar to `*EBE2869D7542FCE37D1C9BBC724B97BDE54428F1`, which is directly stored in the `mysql.user` table by TiDB. To get this string, use `SELECT password('auth_string')` to encrypt your password.
```
Comment (kennytm) [inline sql/privilege.md]: 41-bit → 41-digit or 41-character.

Comment (morgo) [reply sql/privilege.md]: MySQL has ~4 types of hash methods (old passwords, mysql native password, sha256, sha256 caching).  I think it should say something like:

"TiDB supports the `mysql_native_password` authentication mechanism which is the default mechanism used in MySQL 5.7.  Support for additional authentication mechanisms is planned in the future".

You could further say mysql_native_password is based on sha1 if you so desired :-)

**Comment by tiancaiamao** [comment]
LGTM

File: sql/privilege.md [hunk 2]
```diff
@@ -324,10 +324,12 @@ auth_spec: {
...
+- `IDENTIFIED BY 'auth_string'`
 
-  When you set the login password, `auth_string` is encrypted by TiDB and stored in the `mysql.user` table.
+    It is used to set the login password. `auth_string` is encrypted by TiDB and stored in the `mysql.user` table.
 
-- IDENTIFIED BY PASSWORD `hash_string`
+- `IDENTIFIED BY PASSWORD 'hash_string'`
 
-  When you set the login password, `hash_string` is encrypted by TiDB and stored in the `mysql.user` table. Currently, this is not the same as MySQL.
+    It is also used to set the login password. `hash_string` is a 41-character string similar to `*EBE2869D7542FCE37D1C9BBC724B97BDE54428F1`, which is directly stored in the `mysql.user` table by TiDB. To get this string, use `SELECT password('auth_string')` to encrypt your password.
```
Comment (CaitinChen) [inline sql/privilege.md]: 41-character -> 41-bit?
@tiancaiamao @morgo

Comment (kennytm) [reply sql/privilege.md]: @CaitibChen No a bit can only mean a **b**inary dig**it**, it should never be used to describe general strings.

Comment (CaitinChen) [reply sql/privilege.md]: @kennytm  Thanks ^_^

**Pull request merged**